### ISSUE TITLE
[Feature Request] Support for SQF-VM

### ISSUE THREAD

**Issue by X39** [Krzmbrzl/SQDev#179]
https://github.com/SQFvm/vm (reference implementation: https://github.com/SQFvm/debuggerCLI)

There is no support for SQF-VMs debugging interface

**Comment by Krzmbrzl** [comment]
My answer to #178 also applies for this one.

**Comment by X39** [comment]
personally, i do not expect too much work for the SQF-VM due to it using simple network sockets and JSON as data format.

The heavy part is diving into the debugger interface of eclipse

**Comment by Krzmbrzl** [comment]
Yes I think so too. But as I have never touched the whole debugging framework of eclipse I expect this to take quite some time. And I don't want to do it half-heartedly so when I'm doing it, I'll do it properly.
But that will take time...
